versions.tf

```diff
@@ -17,7 +17,6 @@
 terraform {
   required_version = ">=0.12.6"
   required_providers {
-    google      = ">= 3.29, <4.0.0"
-    google-beta = ">= 3.29, <4.0.0"
+    google = ">= 3.49, <4.0.0"
```

Comment:
I believe this is normally considered a breaking change, but I think it should be ok given the recency of the module and the relatively minor bump in version. I'll let @bharathkkb confirm.